doc/building_blocks/neurovault.rst

```diff
@@ -0,0 +1,197 @@
+.. _neurovault:
+
+===========================================================
+Downloading statistical maps from the Neurovault repository
+===========================================================
+
+Neurovault is a public repository of unthresholded statistical maps,
+parcellations, and atlases of the human brain. You can read about it
+and browse the images it contains at http://www.neurovault.org. You
+can download maps from Neurovault with Nilearn.
+
+Neurovault was introduced in [1]_.
+
+Neurovault contains collections of images. We can get information
+about each collection - such as who uploaded it, a link to a paper, a
+description - and about each image - the modality, number of subjects,
+some tags, and more. The nilearn downloaders will fetch this metadata
+and the images themselves.
+
+Nilearn provides two functions to downlad statistical maps from
+Neurovault.
+
+Specific images or collections
+------------------------------
+
+In the simplest case, you already know the "id" of the collections or
+images you want. Maybe you liked a paper and went to
+http://www.neurovault.org looking for the data. Once on the relevant
+collection's webpage, you can click 'Details' to see its id
+(and more). You can then download it using
+``nilearn.datasets.fetch_neurovault_ids`` :
+
+    >>> from nilearn.datasets import fetch_neurovault_ids
+    >>> brainpedia = fetch_neurovault_ids(collection_ids=[1952]) # doctest: +SKIP
```

Comment:
This simple example is fetching me 1325 images. May be we should simple one which fetches least images ?